File: PublicServiceRequest/app/domain/requestFolio.ts
module app.domain {

    export interface IRequestFolio {
        RequestFolioID: number;
        RequestFolioTypeID: number;
        RequestID: number;
        Folio: string;
        FormattedFolio: string;
        Cancelled: string;
        Confidential: string;
        HasOpenPSR: boolean;
        Year: string;
        HistoryRunID: number;
        District: number;
        DORCode: string;
        Owner: string;
        SiteAddress: string;
        SiteCity: string;
        ZipCode: string;
        MailingAddress: string,
        MailingCity: string,
        MailingZipCode: string,
        AdmIsActive: string;
        AdmUserAdded: number;
        AdmUserAddedFullName: string;
        AdmDateAdded: string;
        AdmUserModified: number;
        AdmUserModifiedFullName: string;
        AdmDateModified: string;
        RequestFolioTypeName?: string;
    }

    export class RequestFolio implements IRequestFolio {
        constructor(public RequestFolioID: number,
            public RequestFolioTypeID: number,
            public RequestID: number,
            public Folio: string,
            public FormattedFolio: string,
            public Cancelled: string,
            public Confidential: string,
            public HasOpenPSR: boolean,
            public Year: string,
            public HistoryRunID: number,
            public District: number,
            public DORCode: string,
            public Owner: string,
            public SiteAddress: string,
            public SiteCity: string,
            public ZipCode: string,
            public MailingAddress: string,
            public MailingCity: string,
            public MailingZipCode: string,
            public AdmIsActive: string,
            public AdmUserAdded: number,
            public AdmUserAddedFullName: string,
            public AdmDateAdded: string,
            public AdmUserModified: number,
            public AdmUserModifiedFullName: string,
            public AdmDateModified: string,
            public RequestFolioTypeName?: string) { }
    }
}

File: PublicServiceRequest/Models/RequestFolio.cs
namespace CustomerService_PSR.Models
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    [Table("PSR.RequestFolio")]
    public partial class RequestFolio : BaseClass
    {
        public int RequestFolioID { get; set; }

        public int? RequestID { get; set; }

        public int? RequestFolioTypeID { get; set; }

        [StringLength(25)]
        public string Folio { get; set; }

        [StringLength(25)]
        public string FormattedFolio { get; set; }

        public short? Year { get; set; }

        public int? HistoryRunID { get; set; }

        public int? District { get; set; }

        [StringLength(100)]
        public string DORCode { get; set; }

        [StringLength(200)]
        public string Owner { get; set; }
        [StringLength(500)]
        public string SiteAddress { get; set; }

        [StringLength(500)]
        public string SiteCity { get; set; }

        [StringLength(20)]
        public string ZipCode { get; set; }

        [StringLength(500)]
        public string MailingAddress { get; set; }

        [StringLength(500)]
        public string MailingCity { get; set; }

        [StringLength(20)]
        public string MailingZipCode { get; set; }

        public bool? CancelledAtTimeOfRequestFlag { get; set; }

        public bool? ConfidentialAtTimeOfRequestFlag { get; set; }

        public virtual Request Request { get; set; }

        public virtual RequestFolioType RequestFolioType { get; set; }
    }
}
